litellm/llms/anthropic.py

```diff
@@ -507,16 +507,19 @@ async def make_call(
     model: str,
     messages: list,
     logging_obj,
+    timeout=None,
 ):
     if client is None:
-        client = _get_async_httpx_client()  # Create a new client if none provided
+        client = _get_async_httpx_client({"timeout": timeout} if timeout else None)  # Create a new client if none provided
 
     try:
         response = await client.post(api_base, headers=headers, data=data, stream=True)
```

Comment:
hey @ikbenale you can now pass the timeout in the .post() request - https://github.com/BerriAI/litellm/blob/c6fd2fef9bcafff0b9da4ad2b3787d2d7b7aa989/litellm/llms/custom_httpx/http_handler.py#L83

that should be an easier solution to the problem (will work across passed clients + created clients)

Reply:
@krrishdholakia thanks! Applied that suggestion. I also undid my changes to `litellm/utils.py` since they're no longer necessary under this approach.